tests/test_data/rendered_templates/default.rst

```diff
@@ -0,0 +1,60 @@
+.. _db:
+
+db
+--
+
+Created by H Wierstorf, C Geng, B E Abrougui
+
+============= ======================
+version       `1.0.0 <https://github.com/audeering/db/blob/main/CHANGELOG.md>`__
+license       `CC0-1.0 <https://creativecommons.org/publicdomain/zero/1.0/>`__
+source        https://github.com/audeering/audbcards
+usage         unrestricted
+languages     eng
+format        wav
+channel       1
+sampling rate 8000
+bit depth     16
+duration      0 days 00:00:04
+files         2
+repository    `data-local </tmp/pytest-of-cgeng/pytest-20/test_template_render0/repo/webapp/#/artifacts/browse/tree/General/data-local/db>`__
```

Comment:
This doesn't look right to me as it contains `pytest-of-cgeng`.

Why are the tests passing on Github and locally for me with your username hard coded?

Reply:
This is a duplicate - the assertions are relaxed (using `difflib's` text similarity and the ratio of identical lines)